src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -861,6 +874,43 @@
 )
 
 
+class AlterIndexStatementSegment(BaseSegment):
+    """An `ALTER INDEX` statement.
+
+    https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/ALTER-INDEX.html
+    If possible, please keep the order below the same as Oracle's doc:
+    """
+
+    type = "alter_index_statement"
+    match_grammar: Matchable = Sequence(
+        "ALTER",
+        "INDEX",
+        Ref("IndexReferenceSegment"),
+        OneOf(
+            Sequence(
+                "REBUILD",
+                OneOf(
+                    Sequence("REVERSE"),
+                    Sequence("NOREVERSE"),
+                    optional=True,
+                ),
+            ),
+            Sequence("PARAMETERS", Bracketed(Ref("QuotedLiteralSegment"))),
+            Sequence("COMPILE"),
```

Comment:
These Sequence wrappers aren't needed, if there's nothing else, just write it as `COMPILE`

Reply:
Hi @WittierDinosaur, fix done!